lumigator/backend/backend/tests/integration/api/routes/test_api_workflows.py

```diff
@@ -432,19 +433,49 @@ def test_job_non_existing(local_client: TestClient, dependency_overrides_service
     assert response.json()["detail"] == f"Job with ID {non_existing_id} not found"
 
 
-def wait_for_workflow_complete(local_client: TestClient, workflow_id: UUID):
+def wait_for_workflow_complete(
+    local_client: TestClient,
+    workflow_id: UUID,
+    timeout_seconds: int = 300,
```

Comment:
No defaults at this level, again.

Reply:
The previous (test) code already had magic numbers further down the method (`300` iterations and `1` second sleeps), I don't think this change actually makes things worse as it makes the values and purpose clear in the method signature.

Reply:
Yes, that doesn't mean it was ok :)